components/teams-service/storage/storage.go

```diff
@@ -42,12 +31,11 @@ type Resetter interface {
 
 // Team is the struct ingested and returned by our backend implementations.
 type Team struct {
-	ID          uuid.UUID
-	Name        string
-	Description string
-	CreatedAt   time.Time
-	UpdatedAt   time.Time
-	Projects    []string
+	ID        string
+	Name      string
+	CreatedAt time.Time
+	UpdatedAt time.Time
+	Projects  []string
 }
```

Comment:
`IsTeamDeletable` lower down in the file needs an edit, too.
```
// IsTeamDeletable returns whether or not an input team is deletable by id.
func IsTeamDeletable(teamID string) bool {
	return !stringutils.SliceContains(NonDeletableTeams, teamID)
}
```

Reply:
eep good catch! led me to fix the `AdminTeamName` constant too